ideas.md

```diff
@@ -18,6 +18,8 @@ This is list of ideas for hackathon, feel free to choose one or come up with you
 * Wechat Mini app: login with NEAR and exchange NFTs with other users in the group.
 * NFT for attending meetup: QR code for meetup organizers that allows attendees to scan and claim the specific NFT of the specific event.
 * Marketplace for meetup organizers and speakers. Organizers can create an event and offer slots for speakers. Speakers either use auction mechanic or offer to speak and have attendees vote for who to listen to.
+* Maskbook like plugin:use the API of those social products already have and expand the influence of blockchain by their network. Educate people why privacy matters and what blockchain can do.
+* Dating App: With smart-contract, couples can create some new relationship other than marriage. It is also a use case of DAO.
```

Comment:
hmm let's not try to encourage things that are not protected by law.

Reply:
none of blockchain stuff is really protected by law... which one do you think is sensitive? Maskbook or dating?

Reply:
Maybe I mistunderstood what you mean, but "relationship other than marriage" can be misleading.

Reply:
🤣 of course, it can be what you imagine~ I just imagine that maybe one boy and girl want to have some finality on their relationship but do not want to have a real certification. transparency of the relationship between real close people.

Reply:
hahahahahahaha alright to get this started with being too controversial, let's not mention the relationship other than marriage